packages/fxa-settings/src/components/LinkExpired/index.tsx

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+
+type LinkExpiredProps = {
+  linkType: LinkType;
+};
+
+type LinkType = 'reset-password' | 'signin';
+
+const resendResetPasswordLinkHandler = () => {
+  //   TODO: add resend link action
+};
+
+const resendSigninLinkHandler = () => {
+  //   TODO: add resend link action
+};
+
+const getTemplateValues = (linkType: LinkType) => {
```

Comment:
Not sure how I feel about calling these "template" values, or at least I haven't seen the word "template" used conventionally in React much. You called the other one `headerValues`, does it make more sense to call this `linkValues`?

Reply:
sure! I don't have a strong preference

Reply:
Looked into this closer -- this one is called "template" values because it covers multiple elements and not just the header (or the link). It covers the description text, the header text and also the link text, so I think it works for this instance, but I'll check the others to make sure they're suitable